### PR TITLE
Run docker on forked pull requests

### PR DESCRIPTION
**Pull request by rbren**
This PR ensure docker runs on all pull requests, not just PRs from inside this repo.

It will only `build`, and not `push`--there are some permission errors w/ the latter

### CODE REVIEW

**Comment by rbren** [comment]
Can't figure out the permissions unfortunately--going to un-require this check for now

File: containers/build.sh [hunk 1]
```diff
@@ -3,6 +3,7 @@ set -eo pipefail
 
 image_name=$1
 org_name=$2
+tag_prefix="$3-"
 push=0
 if [[ $3 == "--push" ]]; then
```
Comment (neubig) [inline containers/build.sh:8]: Should this be $4?
```suggestion
if [[ $4 == "--push" ]]; then
```

**Review by neubig — COMMENTED** [review]
This looked basically good to me, I can take another look when tests pass.

File: containers/build.sh (hunk 1 above)
Comment (rbren) [reply containers/build.sh:8]: good catch!

**Review by neubig — APPROVED** [review]
Looks great to me!

**Pull request merged**